docs/analysis-workflows/dna-open-access-filtering.md

```diff
@@ -1,8 +1,47 @@
 ---
 id: dna-open-access-filtering
-title: Open Access Filtering
-sidebar_label: Open Access Filtering
+title: Open Access Variant Filtering
+sidebar_label: Open Access Variant Filtering
 platform_key: DOCS_DNA_PIPELINE
 ---
 
-![Reminder Banner](/assets/submission/banner-reminder.svg)
+The SNV/InDel variant calls from both Sanger & MuTect2 calling results on ARGO portal are further processed through the Open Access Variant Filtering Workflow to create VCF files that can be downloaded by ordinary users from the open-access tier. For details about the process workflow, please see the latest version of the [ARGO Open Access Variant Filtering Workflow](https://github.com/icgc-argo-workflows/open-access-variant-filtering/releases)
```

Comment:
change "ARGO portal" to 
"the ARGO Data Platform"

Also suggest to change "that can be downloaded by ordinary users from the open-access tier." to
"that can be downloaded anonymously from the File Repository."

 (this is how we stated it on a different page, and I find "open access tier" may be a bit confusing)